### PR TITLE
Change color programmatically functionality added

### PR DESCRIPTION
**Pull request by hussamsh**
Hi @JesusM , 

I made 4 changes 

1) A SeekBar should be able to change colors in code, so I added 5 functions that changes colors for different parts of the view in addition to the view's `setBackground(int color)` that covers all the parts of the view.

2) Found some unused allocations, so I removed them

3) updated the README.md to reflect the changes 

4) updated the sample app to showcase changes

cheers,
Hossam

### CODE REVIEW

**Comment by JesusM** [comment]
Hi @hussamsh , thanks for the contribution. I'll take a look to it soon. 
Please, as an advise for future contributions (and when working in environments where you add code via Pull Request to another branch) you should name the branch of the PR with a descriptive name to avoid confusion (the main branch and yours are called `master`).
Best

**Comment by hussamsh** [comment]
@JesusM Sorry for the confusion, I'm still new both to android and git , I will try to make my PR name different from the master branch in the future as I have a couple more ideas for this library 
Thank you.

File: .idea/dictionaries/lenovo.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<component name="ProjectDictionaryState">
```
Comment (JesusM) [inline .idea/dictionaries/lenovo.xml]: Not entirely sure why this is needed.

File: lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java [hunk 2]
```diff
@@ -122,12 +121,13 @@
 	/**
 	 * The pointer's position expressed as angle (in rad).
 	 */
-	private float mAngle;
+    private Paint mArcPaint;
```
Comment (JesusM) [inline lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java]: Both variable don't respect indentation.

File: lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java [hunk 3]
```diff
@@ -447,6 +434,46 @@ public void setValue(float newValue) {
         }
     }
 
+    /**
```
Comment (JesusM) [inline lib/src/main/java/com/jesusm/holocircleseekbar/lib/HoloCircleSeekBar.java]: These methods also respect current indentation spaces. I've seen that `setValue` doesn't respect it so if you can it'd be great if you can fix that.

**Comment by JesusM** [comment]
@hussamsh I've left a few comments only about how code looks, as I can not test it right now. Please, when you have time it'd be great if you can fix them. Could you rebase + force push the code when ready?With that it will be only one commit to review and it would make the review process easier. Thanks!

**Comment by JesusM** [comment]
Closing this because the lack of maintenance from PR's author.

**Pull request closed without merging**